modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/SampleSQLBuilder.java

```diff
@@ -252,14 +289,40 @@ protected Reader generateSQL() {
 						createFileWriter(
 							new File(
 								BenchmarksPropsValues.OUTPUT_DIR,
-								"sample.sql")))) {
+								SampleSQLBuilderConstants.
+									SQL_TEMPLATE_FILE_NAME)))) {
+
+					for (String sqlFileName : _createSQLStatementTemplateList) {
```

Comment:
@slnn If you just merge this part into a separate sql file (like before this change?), you won't need to add https://github.com/liferay-core-infra/liferay-portal/pull/420/commits/95888426a02232d97590df1193f3e4ae1696d9a9, right?

Reply:
Hi @dantewang 

I prefer this one because in the previous changes, I copy so many codes from DBBuilder into SampleSQLBuilder, @tinatian  has pointed out that in the previous review, actually, we can handle these create SQLs'  compression like what we did for these insert sql statements, so I merge these create sql templates and these insert sqls templates that generated by freamarker and then comress them.

Reply:
We just need the DB's buildSQL method please see https://github.com/liferay-core-infra/liferay-portal/commit/95888426a02232d97590df1193f3e4ae1696d9a9, did not need more logic in DBBuilder

Reply:
I  write the create table/index statements into the sampleSQLWriter, because I want to make sure that a sql file's first part must be the create sql statement and then append the inset sql statements or update sql statements

Reply:
@slnn I see...

Reply:
Hi @dantewang 

How about now? If I just need solve the SF issue and resend?

Reply:
@slnn Before you resend, please review your logic to see if they can be clearer. The current logic is a little hard to catch up with...